src/couch_replicator_httpc.erl

```diff
@@ -162,6 +165,9 @@ process_stream_response(ReqId, Worker, HttpDb, Params, Callback) ->
     receive
     {ibrowse_async_headers, ReqId, Code, Headers} ->
         case list_to_integer(Code) of
+        C when C =:= 429 ->
+            maybe_retry(back_off, Worker,
+                HttpDb#httpdb{timeout = ?MAX_BACKOFF_WAIT}, Params);
```

Comment:
Does this change or update the timeout for the duration of this replication's life-cycle.  Wonder if that has any consequence. Or say what happens if 429 errors will start interleaving with other errors (socket timeouts, or authentication ones).